eq-author/src/App/page/Design/CalculatedSummaryPageEditor/AnswerSelector.js

```diff
@@ -248,87 +218,94 @@ export class UnwrappedAnswerSelector extends Component {
                   </AnswerListItem>
                 ))}
               </AnswerList>
-              {minOfTwoAnswersError && <ErrorInline>{errorMsg}</ErrorInline>}
+              {minOfTwoAnswersError && (
+                <ErrorInline>
+                  {buildLabelError(
+                    CALCSUM_ANSWER_NOT_SELECTED,
+                    (unit || answers[0].type).toLowerCase(),
+                    20,
+                    19
+                  )}
+                </ErrorInline>
+              )}
             </ErrorContext>
           </SectionListItem>
         </SectionList>
         <SelectButton
           variant="secondary"
-          onClick={this.handlePickerOpen}
+          onClick={handlePickerOpen}
           data-test="answer-selector"
         >
-          Select another {(answerType || "answer").toLowerCase()} answer
+          Select another {answerType?.toLowerCase() || ""} answer
         </SelectButton>
-      </div>
+      </>
     );
-  }
-
-  renderEmptyState(availableSummaryAnswers) {
-    const { getValidationError } = this.props;
-    const isAvailableAnswers = availableSummaryAnswers.length > 0;
+  };
 
+  const renderEmptyState = (availableSummaryAnswers) => {
     const errorValidationMsg = getValidationError({
       field: "summaryAnswers",
       message: CALCSUM_ANSWER_NOT_SELECTED,
     });
 
-    const title = isAvailableAnswers
+    const title = availableSummaryAnswers.length
```

Comment:
```suggestion
    const [title, text] = availableSummaryAnswers.length  ? ["No answers selected", "Select an answer using the button below."] : ["No answers available", "There are no answers to provide a calculated summary."]
```

Reply:
i have no idea if that's any better or not, but it's an option

Reply:
Yeah the repetition did bug me a bit too. Tried your thing but post-prettierification it doesn't look very nice. I've moved these variables inline into the JSX instead